static/src/javascripts/projects/common/modules/identity/upsell/account-creation.js

```diff
@@ -28,10 +29,18 @@ const bindAccountCreation = (el): void => {
     });
 };
 
+const bindOptouts = (el): void => {
+    fastdom.write(() => {
+        render(<OptOutsList />, el);
+    });
+};
+
 const enhanceAccountCreation = (): void => {
     loadEnhancers([
         ['.js-identity-upsell-account-creation', bindAccountCreation],
+        ['.js-identity-upsell-optputs', bindOptouts],
     ]);
 };
 
+// TODO: generalise this as upsell.js
```

Comment:
I want to do this after @rahilb completes work on the cards so as to not step on each other's toes today, happy to block this PR until we talk about this

Reply:
I'm happy for you to do it in this PR (and merge), I had created my own version of this file to do the binding so i'll incorporate mine into `upsell.js`

Reply:
okay!